lib/utils/azure_storage.ts

```diff
@@ -0,0 +1,48 @@
+import * as azureStorage from "azure-storage";
+import { Option, option } from "ts-option";
+import { Either, left, right } from "./either";
+
+// Storage connection string may be process.env.AzureWebJobsStorage;
+
+export type BlobService = azureStorage.BlobService;
+export type BlobResult = azureStorage.BlobService.BlobResult;
+
+export function getBlobService(connectionString: string): BlobService {
```

Comment:
what value this function adds? it just calls `azureStorage.createBlobService` with the same params, this can be written as:

`export const getBlobService  = azureStorage.createBlobService;`

(same comment as above, why is it needed?)

Reply:
see https://github.com/teamdigitale/digital-citizenship-functions/pull/59#discussion_r141296878